src/core/main.cpp

```diff
@@ -137,6 +142,14 @@ int main( int argc, char * * argv )
 		}
 	}
 
+#ifndef LMMS_WIN32
```

Comment:
> Travis is pissed off at me now.

Im close to turning it off on my repo, that or pay for the osx builds, my inbox is full of travis fail emails.